sympy/physics/mechanics/kane.py

```diff
@@ -697,20 +696,26 @@ def kanes_equations(self, FL, BL):
         Parameters
         ==========
 
-        FL : list
-            Takes in a list of (Point, Vector) or (ReferenceFrame, Vector)
+        loads : iterable
+            Takes in an iterable of (Particle, Vector) or (ReferenceFrame, Vector)
             tuples which represent the force at a point or torque on a frame.
-        BL : list
-            A list of all RigidBody's and Particle's in the system.
-
+            Must be either a non-empty iterable of tuples or None which corresponds
+            to a system with no constraints.
+        bodies : iterable
+            An iterable of all RigidBody's and Particle's in the system.
+            A system must have at least one body.
         """
+        if (bodies is None and loads != None) or isinstance(bodies[0], tuple):
+            # This switches the order if they use the old way.
+            bodies, loads = loads, bodies
+            SymPyDeprecationWarning('The API has changed and will be deprecated, '
+                    'loads is now the second argument and optional.').warn()
```

Comment:
Let's make this more defined:

```python
SymPyDeprecationWarning(value='The API for kanes_equations() has changed such that the loads (forces and torques) are now the second argument and is optional with None being the default.`,
                                            feature='The kanes_equation() argument order',
                                            last_supported_version="1.X.X",
                                            useinstead="switched argument order to update your code, For example: kanes_equations(loads, bodies) > kanes_equations(bodies, loads).",
                                            issue=10XXX,
                                            deprecated_since_version="1.0")
```

We need to open an issue for the deprecation of this argument change and reference in the warning.